### ISSUE TITLE
Xio скрипт и скрипт управления предприятием

### ISSUE THREAD

**Issue by ra81** [cobr123/XioScript#23]
хороший понимаешь скрипт управления предприятием. Вносит фильтры по городам и вообще. Но блин понимаешь вместе они не работают. Например не все везде настроено. И надо выполнить только одну группу юнитов. А хренушки. Либо все магазины, либо офисы. А выбрать город  или регион нельзя. 

Было бы круто если бы сии фильтры просто вкрутили внутрь ксио. польза с них хорошая. Если они будут работать в комбе, будет вообще круто.

**Comment by cobr123** [comment]
есть ссылка на скрипт?

**Comment by ra81** [comment]
да, речь шла про него. Почему я и писал что если овервью как то скрестить с фильтром именно по городам, названию итд. Тогда было бы хорошо. Сейчас я по факту тыкаю каждый нужный юнит руками предварительно наложив фильтр. С вебом сам не могу разобраться. Если стратегии розницы сам сделаю, то здесь увы нужен навык которым я не обладаю.

Что из скрипта можно было бы забрать:
- фильтрацию юнитов
- подсветка красным эффективности меньше 100.
- запрос новой эффективности по клику на цифру эффективности.

Сейчас ксио выводит какую то эффективность персонала которая как бы вообще сама по себе никуда не уперлась. Если выводить общую эффективность, будет полезнее. Ну а эффективность персонала и оборудования можно более мелкими цифрами приписать.
